### ISSUE TITLE
Option to hide or clear user page activity log, followers/following, etc

### ISSUE THREAD

**Issue by atouu** [snarfed/bridgy-fed#1455]
Wouldn't be better if there's an option to hide or clear activity logs in fed.brid.gy/ap? Especially for deleted posts.

**Comment by snarfed** [comment]
Yes! User pages aka activity logs on fed.brid.gy don't have a clear, coherent UX goal or vision yet. They probably need one. Right now they're somewhere between user profile and debugging log, maybe more the latter. Deletes obviously aren't appropriate in user profiles, but they are (more, at least) in debugging logs. Sigh.

Realistically, the most likely approach I see here is #769. @atouu I assume disabling your user page entirely would work for you? If so, I may merge this issue into that one.

**Comment by atouu** [comment]
I'm fine having user page since I want to know if things are working too. But I think disabling it entirely could work for me too.

**Comment by snarfed** [comment]
We disabled paging through historical activities/followers/followees, which helps a bit here.
